components/modal/useModal.js

```diff
@@ -1,10 +1,28 @@
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 
 const useModal = (initialState = false) => {
     const [isOpen, setModalState] = useState(initialState);
-    const open = () => setModalState(true);
-    const close = () => setModalState(false);
     const toggle = () => setModalState(!isOpen);
+    const open = () => setModalState(true);
+    let id;
+
+    const close = () => {
+        const modal = document.querySelector('.pm-modalOverlay');
```

Comment:
What about some very rare cases when there are two popups visible? The one I've seen was when my session ends while a popup is opened, and I get another popup asking me to enter my password again. This selector seems like it might fade out all of the visible popups.

Reply:
![Screenshot 2019-05-09 at 09 51 46](https://user-images.githubusercontent.com/944727/57433073-2d00a080-7240-11e9-9597-b7666d9df109.png)